.github/workflows/build_and_test.yml

```diff
@@ -246,7 +246,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: [ ubuntu-24.04 ]
+        os: [ ubuntu-24.04, macOS-latest ]
```

Comment:
it shouldn't run on everything. 
some of the fuzzers in the list below doesn't have macos test